modules/common-fs2/src/main/scala/com/snowplowanalytics/snowplow/enrich/common/fs2/Enrich.scala

```diff
@@ -130,6 +144,7 @@ object Enrich {
       for {
         etlTstamp <- Clock[F].realTime(TimeUnit.MILLISECONDS).map(millis => new DateTime(millis))
         registry <- enrichRegistry
+        _ <- if (payload.toString.contains("ben-testing")) Timer[F].sleep(2.minutes) else Sync[F].unit
```

Comment:
Don't think the condition gets a lot of `true`s in prod :)

Reply:
Haha yeah it was to test on my pipeline, I forgot to remove it!